CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+
+- ### Avoid acronyms. If using one, make sure to define it on first use.
+
+- ### Favor writing in the [active voice](http://writing.wisc.edu/Handbook/CCS_activevoice.html).
+**Yes:**  The next section includes some basic information about our app release including the app name.
+
+**No:**  In the next section some basic information is included about our app release including the app name.
+
+- ### Favor writing in the [indicative mood](https://grammarist.com/grammar/english-moods/).
+**Yes:**  At runtime, Replicated will rewrite the image tag to pull from `registry.replicated.com`.
+
+**No:**  At runtime, Replicated would rewrite the image tag to pull from `registry.replicated.com`.
+
+- ### Avoid Second Person Pronouns
+**Yes:** A license value can be used to show and hide fields on the settings page.
+
+**No:** You can use your license to allow customer access to hidden settings.
+
+- ### Avoid First Person Pronouns
+**Yes:** The Go playground can be used to validate the regular expression.
+
+**No:** Here I have used the Go playground to validate the regular expression.
+
+- ### Do not reference relative time from when the document was written.
+**Yes:** KOTS 1.15.0 includes a new feature to...
+
+**No:** A new feature of KOTS is...
+
+- ### One sentence per line. It's easier to review and comment on. Markdown will render these properly.
+**Yes:** Do something.
+Do something else.
+
+**No:** Do something. Do something else.
+
+- ### Use relevant, descriptive anchor text in links.
+**Yes:** ```More information is available in the <a href="/link">release notes</a>.```
+
+**No:**  ```For more information, read the release notes <a href="/link">here</a>.```
+
+- ### Only use Markdown backticks on code or command literals.
+The backtick symbol shouldn't be used for emphasis.
+
+- ### Code snippets in bash should not include sudo.
+**Yes:**  ```docker logs -f replicated```
+
+**No:** ```sudo docker logs -f replicated```
+
+- ### Multi-line code snippets should not have the bash prompt.
+**Yes:**
+```
+docker rmi quay.io/replicated/replicated
+docker pull quay.io/replicated/replicated
+```
+**No:**
+```
+$ docker rmi quay.io/replicated/replicated
+$ docker pull quay.io/replicated/replicated
+```
+
+- ### Avoid `<h1>` tags on a page.
+Use`<h2>` tags to separate page content. If there's a need for another `<h1>`, consider a new document.
+
+- ### Do not end header text with a colon.
+**Yes:** `## Installation Guide`
+
+**No:** `## Installation Guide:`
+
+- ### Use hyphens in filenames, not underscores.
+
+- ### When referring to the KOTS product, use all capital letters.
+**Yes:** A KOTS application can include...
+
+**No:** A Kots application can include...
+**No:** A kots application can include...
+
+- ### The product and the company are named Replicated (with a capital R).
+**Yes:** How to install Replicated.
+
+**No:** How to install replicated.
+
+- ### The kotsadm Admin Console is referred to as the Admin Console
```

Comment:
This is the only one I have issue with -- I think kotsadm is fairly succint.

What about kURL? Should we call these "kURL clusters"? "kURL Embedded Clusters"? "KOTS embedded cluster"?

Reply:
My vote is "embedded cluster," since it's the closest to what we communicate in Vendor. You have two installation options with KOTS: "embedded cluster" and "existing cluster." The fact that kURL is used to help us achieve the former is presently a technical detail that we can choose to communicate. 

As another example, I see "Support Bundling" as a feature of KOTS. We can choose to communicate that it's backed by the troubleshoot project, but that's also a technical detail (we haven't been calling it "troubleshoot.sh Support Bundling").

Reply:
@austinchambers  do you want to suggest a change here? or is this good to go?